### ISSUE TITLE
[question] is there a way to pick a default for the prompt " 'c'urrent / 's'el? "

### ISSUE THREAD

**Issue by sinapis** [jarun/nnn#517]
Personally, I always mean selection if it exists, and current if it doesn't. This is a very prompt that happens whenever I perform any action (archive, delete, copy, move, batch rename) after I perform selection.
Is there a way to configure nnn so it will always choose selection and not prompt me?

**Comment by jarun** [comment]
It's not available. I personally use the tools many a time to choose the current entry even if there is a selection. I would suggest you just change the relevant function to:

```c
static int get_cur_or_sel(void)
{
        if (selbufpos)
                return 's'; 

        if (ndents)
                return 'c'; 

        return 0;
}
```

As I explained above, adding an option/config for this is not reasonable enough. When you have a selection and a valid current entry, the most logical thing to do is ask.

**Comment by sinapis** [comment]
I can get your point if a selection exists and it is not visible. However, did you ever made a visible selection and then meant to delete current?

**Comment by 0xACE** [comment]
> However, did you ever made a visible selection and then meant to delete current?

To answer your question. I do this frequently with practically any action. But most commonly when I have items marked and rename files in a folder with `r`.

Personally I'm not a fan of the selection based `r`ename because if displays the full path...

**Comment by jarun** [comment]
> However, did you ever made a visible selection and then meant to delete current?

Yes, I did. And it's not just for remove.

**Comment by jarun** [comment]
@sinapis I have implemented the enhancement at commit 4b4bf6cca55e7f6710113e36c5ff8f776ca045a2 with option `-u`. Please confirm it works as you expect.

**Comment by jarun** [comment]
@sinapis I have removed the `-u` option and `nnn` would always prefer selection over the hovered file. I have been using this for sometime and I do see the benefits of a reduced prompt in my workflows.
